caluma/extensions/visibilities.py

```diff
@@ -60,9 +79,8 @@ def filter_queryset_for_document(self, node, queryset, info):
         )
 
         row_document_on_applicant_form = Q(family__form__is_published=True) | Q(
-            family__work_item__task__slug__in=settings.APPLICANT_TASK_SLUGS
+            family__work_item__task__slug__in=tasks
```

Comment:
I'd say we can leave this as is, because there will never be a need for an applicant to view such row document from a work_item.form that doesn't belong to an applicant task.

Reply:
Reverted